lib/std/Build.zig

```diff
@@ -408,6 +408,22 @@ fn createChildOnly(parent: *Build, dep_name: []const u8, build_root: Cache.Direc
     return child;
 }
 
+/// Returns true if T is *const [_:0]u8
+fn isStringLiteral(comptime T: type) bool {
```

Comment:
I'm new around here so may be off base, but this seems like it would fit it nicely in `std.meta`?   (It could also be moved in a follow on PR, too.)

Reply:
I agree. There's already `std.meta.isZigString`, which handles more types. I'll leave that to a someone else so that this fix can be merged, since I'm not sure what the status is on adding more things to `std.meta`.